spark/dl/src/main/scala/com/intel/analytics/bigdl/example/optimizerWrapper/README.md

```diff
@@ -0,0 +1,60 @@
+## Summary
+This example demonstrates how to use DLEstimator to train the LeNet5 model and then apply transformation on returned Model which is compatible with ML pipeline
```

Comment:
This example demonstrates how to use BigDL with Spark ML pipeline to train and predict LeNet5 model on MNIST dataset.